cpp/ps0008/ps150days.cpp

```diff
@@ -53,110 +46,53 @@ string userInputKey() {
                 isNotDigit = true;
                 break;
             }
+            
+            ++i;
         }
-      
-
     }
-
     return input;
 }

```

Comment:
- [x] prompted user to input a key
- [ ] repeated the prompting process unless the key is valid integer
- [x] removing useless multiples of key value